doc/source/building/build_live_iso.rst

```diff
@@ -1,44 +1,45 @@
 Build an ISO Hybrid Live Image
 ==============================
 
-.. hint::
+.. sidebar:: Abstract
 
-   Make sure you have checked out the example image descriptions
-   For details see :ref:`example-descriptions`
+   This page explains how to build a live image. It contains:
 
-The following example shows how to build a live system image
-based on SUSE Leap
+   * how to build an ISO image
+   * how to start it with QEMU
+   * how to dump the resulting ISO image on an USB stick
 
-.. code:: bash
+In KIWI all generated ISO images are created to be hybrid. This means,
+the image can be used as a CD/DVD or as a disk. This works because
+the ISO image also has a partition table embedded. With more and more
+computers delivered without a CD/DVD drive this becomes important.
 
-    $ sudo kiwi-ng --type iso system build \
-        --description kiwi-descriptions/suse/x86_64/suse-leap-42.1-JeOS \
-        --target-dir /tmp/myimage
+The very same ISO image can be copied onto a USB stick and used as a
+bootable disk.
 
-Find the image with the suffix :file:`.iso` below :file:`/tmp/myimage`.
-For testing the live system a virtual machine with the iso image attached
-as CD/DVD device can be used. The following example shows how to use the
-Qemu system to run the image:
+The following example shows how to build a live ISO image based on
+openSUSE Leap:
 
-.. code:: bash
+1. Make sure you have checked out the example image descriptions,
+   see :ref:`example-descriptions`.
 
-    $ qemu -cdrom LimeJeOS-Leap-42.1.x86_64-1.42.1.iso -m 4096
+2. Build the image with KIWI:
 
-.. _hybrid_iso:
+   .. code:: bash
 
-Use Live System from USB Stick
-------------------------------
+      $ sudo kiwi-ng --type iso system build \
+           --description kiwi-descriptions/suse/x86_64/suse-leap-42.1-JeOS \
+           --target-dir /tmp/myimage
 
-In KIWI all generated iso images are created to be hybrid. This means
-the image can be used as a CD/DVD or as a disk. This works because
-the iso image also has a partition table embedded. With more and more
-computers delivered without a CD/DVD drive this becomes important.
+   Find the image with the suffix :file:`.iso` below :file:`/tmp/myimage`.
+
+3. Test the live image with QEMU:
+
+   .. code:: bash
 
-As a used the very same iso image can be copied onto a USB stick
-and used as a bootable disk. Thus in order to use the above live
-system from stick do the following:
+      $ qemu -cdrom LimeJeOS-Leap-42.1.x86_64-1.42.1.iso -m 4096
 
-1. Plug in a USB stick
+4. Plug in a USB stick
```

Comment:
I don't think we should put the stick deployment as part of this sequence. Actually if you have built the iso and tested it in qemu you are done. Once done you have several other choices what to do with the image

1. burn it on a CD
2. dump it on a USB stick

both actions are performed to be able to store the image on a removable media. These days I would say nobody burns a CD anymore, almost all people will put it on a stick. Thus the stick deployment is worth some words but imho outside of the build and test list. Thoughs ?